src/components/AboutUs/index.jsx

```diff
@@ -1,11 +1,19 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './style.css';
+import React from 'react';
 
 function AboutUs() {
   return (
-    <div className="about-us-container">
-      <h1>About Us</h1>
+    <div className="jumbotron jumbotron-fluid">
+      <div className="container">
+        <h2 className="display-4">About Us</h2>
+        <p className="lead">At QuiglyCare, we understand the importance of compassionate and reliable care for your 
+loved ones. With our comprehensive range of services, we provide a nurturing and 
+supportive environment that enhances their well-being. Explore the possibilities with 
+QuiglyCare and experience personalised care tailored to your unique needs.</p>
+      </div>
```

Comment:
Could you please convert all the text in the components to JSON format within src/data?

Reply:
Hi @ewelinasobora I've made the requested changes for when you have a minute to take a look, thanks

Reply:
yep this looks good

Reply:
Hi @ewelinasobora I've added the additional placeholder images below the jumbotron - 

<img width="789" alt="Screenshot 2024-01-18 204450" src="https://github.com/4codeCrushers/quigly-care/assets/145570648/a43da5f1-226f-4a05-b2bb-3f04a73aeb47">